rpcs3/Json/pad_settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "handlers": {
+    "null": "This controller is disabled and will appear as disconnected to software. Choose another handler to enable it.",
+    "keyboard": "While it is possible to use keyboard as a pad in RPCS3, the use of an actual controller is strongly recommended. To bind mouse movement to a button or joystick, click on the desired button to activate it, then click and hold while dragging the mouse to a direction",
+    "ds3_windows": "To use DualShock 3 handler, you need to install the official DualShock 3 driver first.\nClick <a href=\"https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration\">here</a> for instructions.",
+    "ds3_linux": "To use DualShock 3 handler, you might need to add udev rules to let RPCS3 access the controller.\nClick <a href=\"https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration\">here</a> for instructions.",
+    "ds3_other": "DualShock 3 handler for official DualShock 3 controllers.",
+    "ds4_windows": "If you have any issues with DualShock 4 handler, it might be caused by third-party tools such as DS4Windows. It's recommended that you disable them while using this handler.",
+    "ds4_linux": "To use DualShock 4 handler, you might need to add udev rules to let RPCS3 access the controller.\nClick <a href=\"https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration\">here</a> for instructions.",
+    "ds4_other": "DualShock 4 handler for official DualShock 4 controllers.",
+    "xinput": "XInput handler will work with Xbox controllers and many third-party PC-compatible controllers.",
+    "evdev": "Evdev handler should work with any controller that has linux support.\nIf your joystick is not being centered properly, read <a href=\"https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration\">this</a> for more information.",
+    "mmjoy": "MMJoystick handler should work with almost any controller recognized by Windows. However, it is recommended that you use the more specific handlers if you have a controller that supports them."
+  }
+}

```

Comment:
In general you should never use something like:
Click [here](https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration\") for instructions.

Use something like:
See the official [instructions](https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration\").

Reply:
Doesn't sound quite English yet ;)